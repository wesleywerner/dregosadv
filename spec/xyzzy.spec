purloin wand
wait
gonear valley portal
xyzzy
gonear tomb portal
xyzzy
xyzzy
