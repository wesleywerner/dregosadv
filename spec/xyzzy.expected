  THIS GAME IS IN DEBUG MODE
  
  Dr Ego and the Egg of Man-Toomba
  An Interactive Fiction Story
  by Special Agent.
  Release 1 / Serial number 400410 / Inform v6.33 Library v6.12.3pre SX
  
  The Maro River (in the canoe)
. 
  In the canoe I can also see a trap (which is closed and empty), a fishing
  spear and a fishing net.
  
> >  [Purloined.]
  
> >  I sit back and enjoy the boat ride until...
  
  "Here we are," my guide says as the canoe lands on the shore.
  
  River Shore (in the canoe)
  I can now exit the canoe.
  
  In the canoe I can also see a trap (which is closed and empty), a fishing
  spear and a fishing net.
  
> >  Valley
  Pushing through the jungle I arrive in a small valley, rich with flowering
  orchids that surround me. An unexplored path is to the east. Some stone
  stairs leads upward. An unexplored path is to the south.
  
> >  Nothing happens.
  
> >  Underground Tomb
  The tomb is small yet crammed with all manner of artifacts, there are clay
  pots stacked in each corner, wood-carved idols stand guard between the
  pots. Engravings on the walls dance under the flicker of my torch.
  
  A stone column stands in the center of the room.
  Some stairs leads upward.
  
> >  You are yanked in an unknown direction. You appear in the ...
  
  Valley
  I am in a lush jungle valley full of vibrant orchids. An unexplored path is
  to the east. Some stone stairs leads upward. An unexplored path is to the
  south.
  
> >  You are yanked in an unknown direction. You appear in the ...
  
  Underground Tomb
  The tomb is small yet crammed with all manner of artifacts. Engravings on
  the walls dance under the flicker of my torch.
  
  A stone column stands in the center of the room.
  Some stairs leads upward.
  
> >